core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -173,6 +173,12 @@ Behaviour.specify("#filter-box", '_table', 0, function(e) {
         
         function setEnableWidgetStates() {
             for (var i = 0; i < pluginTRs.length; i++) {
+                var pluginMetadata = pluginTRs[i].jenkinsPluginMetadata;
```

Comment:
So this is just to highlight/remove highlighting things that the user should check, and the case of having an active checkbox is already handled?